src/FeeRouter/RewardReceiver.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+
+interface IArbSys {
+    function withdrawEth(
+        address destination
+    ) external payable returns (uint256);
+}
+
+contract RewardReceiver {
```

Comment:
Maybe a better name would be "ParentChainReceiver" to reflect that this contract receives funds for a parent chain address. We should add natspec comments throughout too.

Reply:
This receives fund from child chain (e.g. nova), and send it to parent chain (e.g. eth mainnet) via withdrawal